packages/core/src/client/bundlerClient.ts

```diff
@@ -42,14 +42,20 @@ export const createBundlerClientFromExisting: <
  * @returns
  */
 export function createBundlerClient<TTransport extends Transport>(
-  args: PublicClientConfig<TTransport, Chain>
+  args: PublicClientConfig<TTransport, Chain> & { type?: string }
```

Comment:
similar to how viem lets setting custom type name right?

Reply:
yea exactly, and lets us reuse this bundler client in other places so we can have the right headers